deps/src/main/java/com/microsoft/azure/sdk/iot/deps/convention/GsonWritablePropertyResponse.java

```diff
@@ -0,0 +1,40 @@
+package com.microsoft.azure.sdk.iot.deps.convention;
+
+import com.google.gson.annotations.SerializedName;
+import lombok.*;
+
+@NoArgsConstructor
+@RequiredArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
```

Comment:
"@ Data" (without the space) lombok annotation should give you most of this for free